Echo/PolySyncGetOpt.cpp

```diff
@@ -0,0 +1,168 @@
+#include "PolySyncGetOpt.hpp"
+#include <algorithm>
+
+using namespace std;
+
+namespace polysync
+{
+
+int PolySyncGetOpt::getOptIdx( const char optret )
+{
+    if( std::find( std::begin( _optChars ), std::end( _optChars ), optret )
+            != std::end( _optChars ) )
+    {
+         // success: container end not reached.
+    }
+    else
+    {
+        return -1;
+    }
+}
+
+
+bool PolySyncGetOpt::optionsParse(const int argc, char *argv[])
```

Comment:
Improvements addressed + completed here, let me know if this is better?

https://github.com/PolySync/PolySync-CPP-Examples/pull/7/commits/81a655e1d1a9ed1b22a1b6684daddeaabfef2a02